### DISCUSSION TITLE
Copy language over to another locale

### DISCUSSION THREAD

**Discussion by zubricks** [payloadcms/payload#4199] [Roadmap]
Provide the ability for users to easily copy content from one locale to another.

**Comment by cbratschi** [comment]
Please add an async filter to allow passing the data to an auto translate service. We use this feature a lot in WPML.

**Reply by r1tsuu** [reply]
Would be nice, i created this plugin https://github.com/r1tsuu/payload-plugin-translator (i will update it and revise the code in some time)
But it's kinda not that trivial working now because i need to iterate only localized fields.

**Comment by creative-andrew** [comment]
Any news on this?

**Comment by denolfe** [comment]
In Progress: https://github.com/payloadcms/payload/pull/8203

**Comment by tkrotoff** [comment]
I guess this can be closed now that https://github.com/payloadcms/payload/pull/8203 has been merged

**Comment by JarrodMFlesch** [comment]
Cleaning up, as said above https://github.com/payloadcms/payload/pull/8203 has been merged!
